docs/deployment/getting_started_locally.md

```diff
@@ -114,6 +114,31 @@ make gardener-up
 make gardener-dev SKAFFOLD_MODULE=gardenlet
 ```
 
+## Debugging Gardener
+
+```bash
+make gardener-debug
+```
+
+This is using skaffold debugging features. In the Gardener case, Go debugging using [Delve](https://github.com/go-delve/delve) is the most relevant use case.
```

Comment:
Maybe it's worth mentioning that the `Cloud Code` integration features even deeper integration with IDEs than running `make gardener-debug` and attaching manually.

Reply:
This is a google cloud specific feature, isn't it? That's at least what the docs say. We don't use Google Services in `make gardener-up` etc, so I would not mention it explicitly here. The information is just one click away anyway, because there is a link to https://skaffold.dev/docs/workflows/debug/